### DISCUSSION TITLE
What do you do with the grub

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [SpiralLinux/SpiralLinux-project#182] [Q&A]
Hellp,
I wonder how can you make grub into such format (openSUSE-like interface)? Is there any patches (I do not find them in files)?
Besides, how to set default kernel?  `GRUB_DEFAULT` not works here.

**Comment by geckolinux** [comment]
Hi there, sorry for the slow reply.

> I wonder how can you make grub into such format (openSUSE-like interface)? 

It has very few modifications to the vanilla Debian defaults, it mainly just included a background image. You can see what gets used in `/config/bootloaders/grub-pc/`
